sys-fs/cryfs/cryfs-0.9.7.ebuild

```diff
@@ -0,0 +1,75 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=5
```

Comment:
Done, but just wondering, why? I looked at the list of EAPI 6 changes and there didn't seem to be anything that was required.